.github/workflows/version.yaml

```diff
@@ -0,0 +1,19 @@
+name: Python Version
+
+on:
+  pull_request:
+    branches: [main]
```

Comment:
Turned this into its own workflow so it can only be triggered on PRs into `main`, rather than dev branches